test_task_1.py

```diff
@@ -0,0 +1,28 @@
+# Our goal is to create a fullname field by concatenating a first and last name
+import unittest
+import task_1
+import random
+
+
+class TestFullNameCreated(unittest.TestCase):
+    def setUp(self) -> None:
+        self.user_records = task_1.create_full_name()
+        self.record_to_test = random.choice(self.user_records)  #avoid wasting time testing every record
```

Comment:
I think this is a bad practice, as it makes the test non deterministic.

Reply:
Thank you for mentioning this, I admittedly put that bit in without being sure it was a good idea, so I'm glad to learn it's not!